packages/components/src/components/flipcard/flipcard.stories.ts

```diff
@@ -50,281 +51,310 @@ export const Default = {
-                }
-              ]
-            }
-          },
-          args,
-          constants: [
-            {
-              type: 'template',
-              name: 'style',
-              value: '<div style="margin-bottom: 40px;">%TEMPLATE%</div>'
-            },
-            {
-              type: 'attribute',
-              name: 'front-variant',
-              value: 'gradient-dark-top'
-            },
-            {
-              type: 'attribute',
-              name: 'back-variant',
-              value: 'gradient-dark-bottom'
-            }
-          ]
-        });
-      })}
-    `;
-  }
-};
 
-/**
- * Use the `base`, `front`, `back`, `front-slot-container`, `back-slot-container`, `front-media`, `back-media`, `front-secondary-gradient` and `back-secondary-gradient` parts to style the flipcard.
- */
-export const Parts = {
-  parameters: {
-    controls: {
-      exclude: [
-        'base',
-        'front',
-        'back',
-        'front-slot-container',
-        'back-slot-container',
-        'front-media',
-        'back-media',
-        'front-secondary-gradient',
-        'back-secondary-gradient'
-      ]
-    }
-  },
-  render: (args: any) => {
-    return generateTemplate({
-      axis: {
-        y: {
-          type: 'template',
-          name: 'sd-flipcard::part(...){outline: solid 2px red}',
-          values: [
-            'base',
-            'front',
-            'back',
-            'front-slot-container',
-            'back-slot-container',
-            'front-media',
-            'back-media',
-            'front-secondary-gradient',
-            'back-secondary-gradient'
-          ].map(part => {
-            return {
-              title: part,
-              value: `<style>#part-${part} sd-flipcard::part(${part}){outline: solid 2px red; outline-offset: -2px;}</style><div id="part-${part}">%TEMPLATE%</div>`
-            };
-          })
-        }
-      },
-      args,
-      constants: [
-        {
-          type: 'template',
-          name: 'style',
-          value: '<div style="margin-bottom: 40px;">%TEMPLATE%</div>'
-        },
-        {
-          type: 'attribute',
-          name: 'front-variant',
-          value: 'gradient-dark-top'
-        },
-        {
-          type: 'attribute',
-          name: 'back-variant',
-          value: 'gradient-dark-bottom'
-        }
-      ]
-    });
-  }
-};
-
-/**
- * `sd-flipcard` is fully accessibile via keyboard.
- */
-
-export const Mouseless = {
-  render: (args: any) => {
-    return html`<div class="mouseless">${generateTemplate({ args })}</div>`;
-  },
-
-  play: async ({ canvasElement }: { canvasElement: HTMLUnknownElement }) => {
-    const el = canvasElement.querySelector('.mouseless sd-flipcard');
-
-    await waitUntil(() => el?.shadowRoot?.querySelector('.flip-card__side--front'));
-
-    el?.shadowRoot?.querySelector<HTMLElement>('.flip-card__side--front')!.focus();
-  }
+export const AspectRatios = {
+  render: () =>
+    html` <div>
```

Comment:
The parent wrapper is unnecessary.
If possible, please adjust the height of the story frame because when I hover on the second flipcard, the browser scrollbar shows up momentarily

Reply:
Updated